### ISSUE TITLE
twisted.runner fails under Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8739]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8739|
|Type|enhancement|
|Created|2016-08-10 00:55:09Z|
|Branch|https://github.com/twisted/twisted/tree/8739-rodrigc-runner-py3|

If I do:

tox -e py35-tests twisted.runner

it fails

<details><summary>Searchable metadata</summary>

```
trac-id__8739 8739
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8739_rodrigc_runner_py3 8739-rodrigc-runner-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470790509464000 1470790509464000
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/481

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset d425bbe37b13d87600a6cf28ee8f9e1dcfec4cda
```
#!CommitTicketReference repository="" revision="d425bbe37b13d87600a6cf28ee8f9e1dcfec4cda"
Merge pull request #481 from twisted/8739-rodrigc-runner-py3

Author: rodrigc
Reviewer: alex
Fixes: #8739

Port twisted.runner to Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
